src/main/kotlin/com/geekbeast/authentication/JwtTokenGenerator.kt

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2019. OpenLattice, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * You can contact the owner of the copyright at support@openlattice.com
+ *
+ *
+ */
+
+package com.geekbeast.authentication
+
+import com.openlattice.authentication.Auth0AuthenticationConfiguration
+import com.openlattice.authentication.Auth0Configuration
+
+/**
+ *
+ * @author Matthew Tamayo-Rios &lt;matthew@openlattice.com&gt;
+ */
+class LocalJwtTokenProvisioner (
+        private val configurations: Set<Auth0AuthenticationConfiguration>,
+        private val usersAndRoles : Map<String, Set<String>>) {
+
+//    fun
+//    fun buildJwt( String signingAlgorithm, principalId : String, roles : Set<String> ) {
```

Comment:
is this needed or safe to remove?

Reply:
yeah, it's not used in the other PRs neither